src/StkIface.cpp

```diff
@@ -197,4 +197,9 @@ void StkIface::init_vars()
     }
 }
 
+bool StkIface::get_hole_map_algorithm()
+{
+	return tg_->get_hole_map_algorithm();
```

Comment:
what's happening here?

Reply:
`use_adaptive_hole_map_` is a private member of `TiogaSTKIface`. It was defined there since `use_ngp_iface` was also defined as such. Here, `tg_` is a `TiogaSTKIface` instance inside `StkIface`, and it is a private member of StkIface. So, the variable `use_adaptive_hole_map_` has to be accessed through a function that is defined in `StkIface` since we are not directly dealing with the `TiogaSTKIface `class.